stock_request/models/stock_request.py

```diff
@@ -399,12 +399,14 @@ def _action_launch_procurement_rule(self):
                 continue
 
             # If stock is available we use it and we do not execute rule
+            # We need to use the free_qty field because it is the one that has the
```

Comment:
No need of this explanation in code, as the field is self-explanatory. The comment is for the commit message.